src/rank_llm/analysis/response_analysis_verbose.py

```diff
@@ -120,17 +162,35 @@ def count_errors(
 
 
 def main(args):
-    response_analyzer = ResponseAnalyzer(args.files)
-    responses, num_passages = response_analyzer.read_saved_responses()
-    print("Normalized scores:")
+    if args.files:
+        input_data = args.files
+    elif args.input_json_analyzer:
+        input_data = load_json_input_analyzer(args.input_json_analyzer)
+    else:
+        raise ValueError("Either --files or --input_json must be provided.")
+
+    response_analyzer = ResponseAnalyzer(input_data)
+    responses, num_passages = response_analyzer.read_responses()
 
-    print(response_analyzer.count_errors(responses, num_passages, args.verbose))
     # Print normalized scores
+    print("Normalized scores:")
+    print(response_analyzer.count_errors(responses, num_passages, args.verbose))
```

Comment:
Please change count_errors to call read_responses() so that the public caller only calls count_errors after constructing the response analyzer object